web/src/search/results/SearchResults.tsx

```diff
@@ -63,7 +63,7 @@ export interface SearchResultsProps
     deployType: DeployType
     setVersionContext: (versionContext: string | undefined) => void
     availableVersionContexts: VersionContext[] | undefined
-    lastVersionContextKey: string
+    lastVersionContextKey: string | null
```

Comment:
Don't know if "name" is the right term, but "key" seems wrong now?
```suggestion
    lastVersionContextName: string | null
```